### ISSUE TITLE
Error when building the Webpack DLL...

### ISSUE THREAD

**Issue by gabrielrtakeda** [mxstbr/react-boilerplate#658]
When building webpack dll, occurs the error below:

```
npm run build:dll                                                                                                                                                                

> react-boilerplate@3.0.0 build:dll /Library/WebServer/Documents/htdocs/frontend.dev/react/react-boilerplate
> node ./internals/scripts/dependencies.js

Building the Webpack DLL...
/Library/WebServer/Documents/htdocs/frontend.dev/react/react-boilerplate/node_modules/babel-preset-es2015-webpack/index.js:35
    throw new Error('Cannot remove "babel-plugin-transform-es2015-modules-commonjs" from the plugin list.');
    ^

Error: Cannot remove "babel-plugin-transform-es2015-modules-commonjs" from the plugin list.
    at Object.<anonymous> (/Library/WebServer/Documents/htdocs/frontend.dev/react/react-boilerplate/node_modules/babel-preset-es2015-webpack/index.js:35:11)
    at Module._compile (module.js:425:26)
    at Module._extensions..js (module.js:432:10)
    at Object.require.extensions.(anonymous function) [as .js] (/Library/WebServer/Documents/htdocs/frontend.dev/react/react-boilerplate/node_modules/babel-register/lib/node.js:166:7)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at /Library/WebServer/Documents/htdocs/frontend.dev/react/react-boilerplate/node_modules/babel-register/node_modules/babel-core/lib/transformation/file/options/option-manager.js:391:22
    at Array.map (native)
```

It seems to be an error produced by `babel-preset-es2015-webpack` module.
Resolved in this issue: https://github.com/gajus/babel-preset-es2015-webpack/issues/10

Just run `npm uninstall --save-dev babel-preset-es2015 ` and thats be ok.

Sorry for the bad English.

**Comment by mxstbr** [comment]
I thought we used `babel-preset-es2015-webpack` anyway?

**Comment by Antonhansel** [comment]
Same error here:
I've used the project a few times before, even last week without any issues.
Same issue now after a `git clone` and then a `npm run setup`.
Node: 5.0.0
Npm: 3.10.3

**Comment by lucasqiu** [comment]
@Antonhansel 
got same error too. I believe it is due to this recent commit while eslint-plugin-import ditched 1.10.1 and bumped to 1.10.2.
https://github.com/mxstbr/react-boilerplate/commit/a279b0ece6d65c3c3e8870f9a89d3a43d7bf8928
https://github.com/benmosher/eslint-plugin-import/commit/21be81c0feb4b2008c0fba4ca62a50a580c0e779

@mxstbr I tried using eslint-plugin-import@1.10.2 and worked. Let me know if you think that sounds like a good resolution.

**Comment by gihrig** [comment]
Merged #660 bump eslint-plugin-import to 1.10.2 smooth setup. Should work, Try now.

> ...
> Building the Webpack DLL...
> Hash: 4d0224339b8b0e0be27e
> Version: webpack 2.1.0-beta.13
> Time: 6006ms
> ...

Mac OS 10.11.5

Thanks for the PR @lucasqiu 😄
